tests/test_mibs.py

```diff
@@ -30,3 +35,16 @@ def test_init_sync_d_lag_tables(self):
 
         self.assertTrue("PortChannel_Temp" in lag_name_if_name_map)
         self.assertTrue(lag_name_if_name_map["PortChannel_Temp"] == [])
+
+    @mock.patch('swsssdk.dbconnector.SonicV2Connector.get_all', mock.MagicMock(return_value=({})))
```

Comment:
>swsssdk [](http://example.com/codeflow?start=17&length=7)

swsssdk or swsscommon?

Reply:
Did you run `python3 -m pytest -s -v -k test_init_sync_d_interface_tables` ?

Reply:
swsssdk is the right one (method get_all is part of class SonicV2Connector which is implemented in sonic-buildimage/sonic-py-swsssdk/src/swsssdk/dbconnector.py).
Yes, I ran python3 -m pytest -s -v -k test_init_sync_d_interface_tables.